api/proxy/common/eigenda_network.go

```diff
@@ -27,6 +30,8 @@ func (n EigenDANetwork) GetEigenDADirectory() (string, error) {
 // GetServiceManagerAddress returns, as a string, the address of the EigenDAServiceManager contract for the network.
 func (n EigenDANetwork) GetServiceManagerAddress() (string, error) {
 	switch n {
+	case MainnetEigenDANetwork:
```

Comment:
Add todo to get rid of this function and fetch from directory instead

Reply:
[f70eff2](https://github.com/Layr-Labs/eigenda/pull/1756/commits/f70eff23b7f654d475b137e4afacc96114a18f2f)